assets/style.css

```diff
@@ -1,54 +1,50 @@
 /* CSS styling elements go here */
 :root {
-    --soft-white: #fcfdf4;
-    /* no seriously, daylight has a known color, this should be close */
-    --daylight: #fcfcff;
-    --lavender: #e4e0fd;
-    --lime: #5af7a2;
-    --coral: #fbbdbd;
-    --straw: #fff8ea;
-    --orange-smoothie: #ffccae;
-    --slate-blue: #a9b3cc;
-    --deep-scarlet: #e80040;
-    --primary-green: #00d1b2;
-    --pale-indigo: #e5e7fa;
+  --soft-white: #fcfdf4;
+  /* no seriously, daylight has a known color, this should be close */
+  --daylight: #fcfcff;
+  --straw: #e0e0e0;
+  --slate-blue: #a9b3cc;
+  --deep-scarlet: #e80040;
+  --primary-green: #00d1b2;
+  --aluminum: #d8d8d8;
+  /* --orange-smoothie: #ffccae; */
+  /* --lavender: #e0e0e0; */
+  /* --lime: #5af7a2; */
+  /* --coral: #fbbdbd; */
+
 }
 
 html {
-    background-color: var(--daylight);
-    color: black;
-    /* these settings may change depending on how things look */
-    /* width: 100%; */
-    margin-left: 2%;
-    margin-right: 2%;
+  background-color: var(--daylight);
+  color: black;
 }
 
 .hidden {
-    visibility: hidden;
+  visibility: hidden;
 }
 
 body {
-    background-color: var(--daylight);
-    /* width: 100%; */
+  background-color: var(--daylight);
 }
 
 header {
-    width: 100%;
-    text-align: inherit;
+  width: 100%;
+  text-align: inherit;
 }
 
 /* we likely own't even need this */
```

Comment:
if it's not needed, remove it

Reply:
I am happy to do that in re-factoring it and often do not know these things until later